### ISSUE TITLE
maxwell reading old data from bin log

### ISSUE THREAD

**Issue by aashu-agg-28** [zendesk/maxwell#1371]
We are using maxwell to read data from AWS Aurora and push to kinesis. This was working fine so far.

Recently by mistake, a team member ran an update which updated all of the records in tables which was around 3 million. It created a huge load on maxwell and kinesis consumer. So to stop it, we cleared all the tables of maxwell, and restarted it. It reads latest record for sometime, after which it again starts reading old records only.

binlog retention period in Aurora is null, which means log are purged as soon as possible. How do we reset maxwell to read only new incoming updates and not previous ones. Please help.

**Comment by osheroff** [comment]
`drop database maxwell` should do it. clearing the tables (especially `maxwell.positions` should also do it, I'm not sure why your first try didn't work.)

**Comment by osheroff** [comment]
closing the issue, please update with any new info

**Comment by aashu-agg-28** [comment]
Just want to update with finding, may help someone else as well. 

So we dropped the positions table and ran an update query. Started maxwell after update query finished, still maxwell was getting the records from the update query. So looks like it hooked to binlog not from the position maxwell was re-started but from some previous position. After we cleaned all the older binlogs and dropped and started maxwell. It started working well.
